h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
We kicked ourselves the moment we allowed ourselves to implement this initially with a solution which was not complete and without a sufficiently motivating use case, user request, or any initial research.